### PR TITLE
Fix types of wasm builtins.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by leonardoalt** [comment]
Needs to be rebased for js reasons

File: libyul/backends/wasm/WasmDialect.cpp [hunk 1]
```diff
@@ -46,12 +46,22 @@ WasmDialect::WasmDialect():
...
+	m_functions["i64.ge_u"_yulstring].returns.front() = "i32"_yulstring;
+	m_functions["i64.eq"_yulstring].returns.front() = "i32"_yulstring;
+	m_functions["i64.ne"_yulstring].returns.front() = "i32"_yulstring;
+
 	addFunction("i64.eqz", 1, 1);
+	m_functions["i64.eqz"_yulstring].returns.front() = "i32"_yulstring;
 
 	addFunction("i64.store", 2, 0, false);
+	m_functions["i64.store"_yulstring].parameters.front() = "i32"_yulstring;
 	m_functions["i64.store"_yulstring].sideEffects.invalidatesStorage = false;
```
Comment (axic) [inline libyul/backends/wasm/WasmDialect.cpp:61]: Storage?!

Comment (chriseth) [reply libyul/backends/wasm/WasmDialect.cpp:61]: `i63.store` writes to memory, and this line registers that it does not modify storage.

**Comment by chriseth** [comment]
Rebased.

**Comment by chriseth** [comment]
@axic oh sorry, did I overwrite your commit?

**Comment by axic** [comment]
> oh sorry, did I overwrite your commit?

If anything I was just rebasing IIRC.

File: libyul/backends/wasm/WasmDialect.cpp (hunk 1 above)
Comment (axic) [reply libyul/backends/wasm/WasmDialect.cpp:61]: Sorry, mistook false for true, haha :)

**Review by axic — APPROVED** [review]

**Pull request merged**